SonarQube.TeamBuild.PreProcessor/Roslyn/RoslynAnalyzerProvider.cs

```diff
@@ -18,13 +18,17 @@ namespace SonarQube.TeamBuild.PreProcessor.Roslyn
 {
     public class RoslynAnalyzerProvider : IAnalyzerProvider
     {
-        public const string RoslynCSharpFormatName = "roslyn-cs";
-        public const string RoslynCSharpRulesetFileName = "SonarQubeRoslyn-cs.ruleset";
+        public const string RoslynFormatNamePrefix = "roslyn-{0}";
+        public const string RoslynRulesetFileNamePrefix = "SonarQubeRoslyn-{0}";
```

Comment:
@dbmeneses the `.ruleset` part is missing, shoud be `SonarQubeRoslyn-{0}.ruleset`